tips/templates/tips/detail.html

```diff
@@ -0,0 +1,3 @@
+Posted by {{tip.author}} at {{tip.date}} in {{ tip.category }}
+<h1>{{ tip.text }}</h1>
+<li><a href="/tips/">return</a></li>
```

Comment:
I think you should put all the tips in one place and not as link 
the person looking at that links list have no idea what every tip is

lets set up all the tips in a sort of gallery of tips like a notes gallery

Reply:
The idea is to display the tips content in a different page and in the main page show either a short snippet of the text or a separate title

Reply:
![image](https://user-images.githubusercontent.com/34008861/163440868-3eba0e1c-9155-4906-8008-fe1154803c9d.png)

Reply:
I agree with Lior
As far as I see the tips model doesn't have a "short snippet" of the text as you said.
If we want one, we need to add it to the model and also let the user add that data in the form
I think that the view here should be some kind of an Infinite Scroll
What do you think about this idea?

Reply:
It's a good idea, I'll look for a good way to add it

Reply:
It's a good idea, I'll look for a good way to add it

Reply:
comments about visual design should move here https://github.com/redhat-beyond/Xpense/pull/78